source/reference/verification/hash.txt

```diff
@@ -0,0 +1,99 @@
+.. _c2c-verify-hash-comp:
+
+===========================
+Verify with Hash Comparison
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can verify sync by comparing MD5 hashes of collections synced from the
```

Comment:
I think we should note that this will be slower than document counts.

Reply:
Fixed